src/libsyntax/codemap.rs

```diff
@@ -1274,4 +1340,45 @@ mod tests {
 
         assert_eq!(sstr, "blork.rs:2:1: 2:12");
     }
-}
+
+    #[test]
+    fn t10() {
+        //Test span_to_expanded_string works in base case (no expansion)
+        let cm = init_code_map();
+        let span = Span {lo: BytePos(0), hi: BytePos(12), expn_id: NO_EXPANSION};
+        let sstr = cm.span_to_expanded_string(span);
+
+        assert_eq!(sstr, format!("{}\n{}", cm.span_to_snippet(span).unwrap(),
+                                 cm.span_to_string(span)));
```

Comment:
I would just use a string literal, rather than constructing the string with `format!`